guilib/measure/controller/abc.py

```diff
@@ -468,10 +468,10 @@ def time_to_trigger(self):
     def _update_serial_from_settings(self):
         """Set serial settings from new controller settings."""
         serial_cls_name = self.settings.get('controller', 'serial_class',
-                                             fallback='')                       # TODO: #<242> (remove fallback in 1.0)
-        if not serial_cls_name:                                                 # TODO: #<242> (only here for backwards compatibility, remove in 1.0)
+                                            fallback='')                       # TODO: #242 (remove fallback in 1.0)
+        if not serial_cls_name:                                                 # TODO: #242 (only here for backwards compatibility, remove in 1.0)
```

Comment:
Once we have the aliases in place as in #242, I think we can leave this part in. I would remove the "remove XXX in 1.0"